ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -703,6 +703,7 @@ Loader {
                     }
                 }
 
+                hasLinks: linkUrlsModel.count
```

Comment:
linkUrlsModel.count > 0

Reply:
The same thing effectively, as they both evaluate to a truthy value